test/AppLifecycle/AppLifecycle.vcxitems

```diff
@@ -19,11 +19,13 @@
   </ItemGroup>
   <ItemGroup>
     <CopyFileToFolders Include="$(TargetDir)..\AppLifecycleTestApp\AppLifecycleTestApp.exe" DestinationFolders="$(TargetDir)" TreatOutputAsContent="true" />
-    <!-- 
+    <CopyFileToFolders Include="$(MSBuildThisFileDirectory)AppLifecycle-AppxManifest.xml" DestinationFolders="$(TargetDir)" TreatOutputAsContent="true" />
```

Comment:
Consider $(MSBuildProjectDirectory) instead or really any of the more specific to the project and not the current target file options.

https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-reserved-and-well-known-properties?view=vs-2019

Reply:
Not a bad idea but I'd followed other snippets in Reunion (in this case `dev\MRTCore\mrt\Core\unittests\UnitTest.vcxproj` and `dev\MRTCore\mrt\mrm\UnitTests\BaseUnitTests.vcxproj`). Since it's working right now, it's consistent with MRT and it was a real bear to get working correctly at first I'll defer this to a future motivated developer. Not like we won't be in the .vcxproj files again :P